.github/workflows/macos_installation_instructions.yml

```diff
@@ -0,0 +1,51 @@
+name: Verify MacOS Installation Instructions
+# The file macos_installation_instructions.yml is generated with the script scripts/macos-instructions.sh
+on:
+  schedule:
+    - cron: '30 12 * * 1' #  This is every Monday at 5:30am PST/8:30am EST
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout dockstore/docktore
+        uses: actions/checkout@v3
+        with:
+          repository: dockstore/dockstore
+          path: dockstore
+      - name: Checkout dockstore/docktore-ui2
+        uses: actions/checkout@v3
+        with:
+          repository: dockstore/dockstore-ui2
+          path: dockstore-ui2
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        working-directory: dockstore
+      - run: git
+        working-directory: dockstore
+        continue-on-error: true
```

Comment:
if the exit code does not matter, could also consider `git || true` or `git --version`
I think either of those would avoid the distracting build error annotation on the bottom of  https://github.com/dockstore/dockstore/actions/runs/2792043132

Reply:
I have changed it so the command is now,
```
    - code:
        run: git || true
        working-directory: dockstore
```
I have also made it so if you enter ` || true` into `macos_instructions.yml` ` || true` will not be shown to the user.